utbot-framework/src/main/kotlin/org/utbot/framework/plugin/api/TestCaseGenerator.kt

```diff
@@ -116,9 +121,72 @@ open class TestCaseGenerator(
         if (UtSettings.testMinimizationStrategyType == TestSelectionStrategyType.DO_NOT_MINIMIZE_STRATEGY) {
             executions
         } else {
-            minimizeTestCase(executions) { it.result::class.java }
+            minimizeTestCase(executions.filterCoveredInstructions()) { it.result::class.java }
         }
 
+    private fun List<UtExecution>.filterCoveredInstructions(): List<UtExecution> {
+        // Do nothing when we launched not from IDEA
+        if (buildDirs.isEmpty()) return this
+
+        val entityAnnotation = utContext.classLoader.tryLoadClass("javax.persistence.Entity")
```

Comment:
Make it more readable and also easier to edit
```suggestion
        val ignoreCoverageAnnotations = listOfNotNull(utContext.classLoader.tryLoadClass("javax.persistence.Entity")
```